src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
```

Comment:
Can we please change it to,

:data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-summary-info` : 'ec-summary-info"

We need to have it like that for every new component we create. It allows extra data-tests to be applied other than the  default one

Reply:
ok corrected